### PR TITLE
*boops your nose*

### PR DESCRIPTION
**Pull request by yorii**
## About The Pull Request

Ports nose booping from https://github.com/CHOMPstation/CHOMPstation/pull/638 with minor changes.

## Why It's Good For The Game

Roleplaying mechanics

## Changelog
:cl:
add: Added nose boops
/:cl:

### CODE REVIEW

**Comment by Naevii** [comment]
Speedmerge before some intellectual call it a fetish content

**Comment by DatBoiTim** [comment]
> Speedmerge before some intellectual call it a fetish content 

This is ERP. Firm No

**Comment by zeskorion** [comment]
> Speedmerge before some intellectual call it a fetish content

Not fetish content just cute

**Comment by USB3pt0** [comment]
Cannot boop Ian.  Pointless PR.

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -281,6 +281,13 @@
 				SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/betterhug, M)
 		for(var/datum/brain_trauma/trauma in M.get_traumas())
 			trauma.on_hug(M, src)
+	else if(M.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		playsound(get_turf(src), 'sound/items/toysqueak1.ogg', 50, 1, -1)
+		var/nose_type = isipc(M) ? "screen" : "nose"
```
Comment (francinum) [inline code/modules/mob/living/carbon/carbon_defense.dm:286]: Likely to get cluttered. generalize to a var on species.

Comment (yorii) [reply code/modules/mob/living/carbon/carbon_defense.dm:286]: How do you mean, in the species datum?

**Comment by ghost** [comment]
make it possible to boop Ian

**Comment by utilbird** [comment]
> Speedmerge before some intellectual call it a fetish content

Citadel-Station-13/Citadel-Station-13#10204
(which was a vore codebase port :flushed:)

**Review by utilbird — CHANGES_REQUESTED** [review]
Unlabled port, credit the original PR. Code is a bit different but the basis is close enough.
(messages are word for word too)

**Comment by PowerfulBacon** [comment]
I hope your not actually considering this

**Comment by yorii** [comment]
> Unlabled port, credit the original PR. Code is a bit different but the basis is close enough.
> (messages are word for word too)

Thanks, I honestly didn't know where it was originally from.

**Comment by Effemu** [comment]
it's a fetish content

File: code/modules/mob/living/carbon/carbon_defense.dm (hunk 1 above)
Comment (ToasterBan) [reply code/modules/mob/living/carbon/carbon_defense.dm:286]: wouldn't lizards have snouts?

**Comment by ToasterBan** [comment]
how the fuck is touching someone's nose fetish content? do you get off to people touching noses?

call it cringe not fetish content

**Comment by PowerfulBacon** [comment]
🔑

**Comment by yorii** [comment]
I found the intellectuals.

**Pull request closed without merging**